integration-tests/spring-data-jpa/src/main/java/io/quarkus/it/spring/data/jpa/SongResource.java

```diff
@@ -22,6 +25,16 @@ public SongResource(SongRepository songRepository) {
     @Path("/all")
     public List<Song> all() {
         Pageable wholePage = Pageable.unpaged();
-        return songRepository.findAll();
+        Page<Song> all = songRepository.findAll(wholePage);
+        return all.getContent();
```

Comment:
I think we need another method that return the make sure that the result  of `songRepository.findAll(wholePage);` is what we expect it to (that the page doesn't have a previous or next page).
Something like: https://github.com/quarkusio/quarkus/blob/master/integration-tests/spring-data-jpa/src/main/java/io/quarkus/it/spring/data/jpa/PersonResource.java#L135